notebook/notebookapp.py

```diff
@@ -1156,7 +1157,7 @@ def init_server_extensions(self):
             if not modulename in self.nbserver_extensions:
                 self.nbserver_extensions[modulename] = True
         
-        for modulename in self.nbserver_extensions:
```

Comment:
Nevermind, I see that `server_extensions` is a list, so it's already a stable order.